tensorrt_llm/llmapi/llm_args.py

```
@@ -2340,6 +2340,55 @@ def validate_histogram_buckets(cls, v: Optional[List[float]],
+        "Number of batch size sample points for each decode context length.",
+        status="prototype")
+
+    warmup_iterations: NonNegativeInt = Field(
+        default=5,
+        description=
+        "Number of synthetic warmup iterations to run before collecting "
+        "self-benchmark results.",
+        status="prototype")
+
+    output_path: str = Field(
+        default="/tmp/trtllm_self_benchmark.json",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/trtllm_self_benchmark.json"

(S108)